### DISCUSSION TITLE
PoreSpy works fine but cannot display images

### DISCUSSION THREAD

**Discussion by Xiaozizhuo** [PMEAL/porespy#665] [Q&A] [answered]
Hello, dear professor. I am a beginner with pores. Today I tried to run the code in the examples for the first time and I found that all the examples work fine, no errors are reported, but no images are generated, why is that? Is it because there is Chinese on this road?
![1](https://user-images.githubusercontent.com/98071787/160237071-329628ab-65a2-4ee0-976f-50651d7b10e7.png)
![2](https://user-images.githubusercontent.com/98071787/160237075-7bc94583-5f76-4c92-9bdb-e5e023904dd3.png)

**Comment by ma-sadeghi** [comment, marked as answer]
If you're running the examples/scripts in VSCode, try adding `plt.show()` to the last line of the script.

**Reply by Xiaozizhuo** [reply]
Thank you for your helpful help, he was up and running
